<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTopSecret2</class>
 <widget class="QWidget" name="FormTopSecret2">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Segurança do Sistema</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qdocbase.qrc">
    <normaloff>:/images/png/images/docbase64x64.png</normaloff>:/images/png/images/docbase64x64.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_Operators">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Operadores</string>
     </property>
     <widget class="QLabel" name="label_Oper_Pass">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Operator">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_Oper_User">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuário</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Operator_Password">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_Oper_Repeat_Pass">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Operator_PassRepeat">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>331</y>
        <width>611</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>238</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Save">
         <property name="text">
          <string>&amp;Gravar [F10]</string>
         </property>
         <property name="shortcut">
          <string>F10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Update">
         <property name="text">
          <string>&amp;Atualizar [F9]</string>
         </property>
         <property name="shortcut">
          <string>F9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Delete">
         <property name="text">
          <string>Apagar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView_Operators">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>611</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Message_01">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duplo-clique no nome do operador para editá-lo</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Administrator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>191</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>191</height>
      </size>
     </property>
     <property name="title">
      <string>Administrador</string>
     </property>
     <widget class="QLabel" name="label_UserName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Password">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Password">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Release">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Liberar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/cadeado_aberto32x32.png</normaloff>:/cadeado_aberto32x32.png</iconset>
      </property>
     </widget>
     <widget class="Line" name="line_01">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>20</width>
        <height>161</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_ChangeAdmPassword">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>26</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Trocar Senha do Administrador</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_CurrentPass">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>61</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Senha Atual</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CurrentPass">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>61</y>
        <width>109</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NewPass">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>109</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ConfirmPass">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>119</y>
        <width>109</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_AdmRepeatPass">
      <property name="geometry">
       <rect>
        <x>272</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Repetir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_UpdateAdm">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Atualizar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NewPass">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>91</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Senha Nova</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_UserName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <zorder>label_NewPass</zorder>
     <zorder>label_UserName</zorder>
     <zorder>label_Password</zorder>
     <zorder>lineEdit_Password</zorder>
     <zorder>pushButton_Release</zorder>
     <zorder>line_01</zorder>
     <zorder>label_ChangeAdmPassword</zorder>
     <zorder>label_CurrentPass</zorder>
     <zorder>lineEdit_CurrentPass</zorder>
     <zorder>lineEdit_NewPass</zorder>
     <zorder>lineEdit_ConfirmPass</zorder>
     <zorder>label_AdmRepeatPass</zorder>
     <zorder>pushButton_UpdateAdm</zorder>
     <zorder>lineEdit_UserName</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_UserName</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>pushButton_Release</tabstop>
  <tabstop>lineEdit_CurrentPass</tabstop>
  <tabstop>lineEdit_NewPass</tabstop>
  <tabstop>lineEdit_ConfirmPass</tabstop>
  <tabstop>pushButton_UpdateAdm</tabstop>
  <tabstop>lineEdit_Operator</tabstop>
  <tabstop>lineEdit_Operator_Password</tabstop>
  <tabstop>lineEdit_Operator_PassRepeat</tabstop>
  <tabstop>tableView_Operators</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Update</tabstop>
  <tabstop>pushButton_Delete</tabstop>
 </tabstops>
 <resources>
  <include location="qdocbase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Release</sender>
   <signal>clicked()</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>pushButton_Release_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_UpdateAdm</sender>
   <signal>clicked()</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>pushButton_UpdateAdm_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView_Operators</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>tableView_Operators_doubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Save</sender>
   <signal>clicked()</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>pushButton_Save_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Update</sender>
   <signal>clicked()</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>pushButton_Update_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Delete</sender>
   <signal>clicked()</signal>
   <receiver>FormTopSecret2</receiver>
   <slot>pushButton_Delete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushButton_Release_clicked()</slot>
  <slot>pushButton_UpdateAdm_clicked()</slot>
  <slot>pushButton_Save_clicked()</slot>
  <slot>pushButton_Update_clicked()</slot>
  <slot>pushButton_Delete_clicked()</slot>
  <slot>tableView_Operators_doubleClicked(QModelIndex)</slot>
 </slots>
</ui>
